bin/flatsheet

```diff
@@ -57,27 +56,25 @@ function account () {
         }
       };
 
-      accounts.create(result.username, opts, function (err) {
-
-        //todo: notification of error
+      flatsheet.accounts.create(result.username, opts, function (err) {
         if (err) console.error(err);
-        db.close();
+        process.exit()
```

Comment:
aaahh, you're right. i was taking a shortcut with process.exit. pushing a commit that uses flatsheet.db.close instead